batch/create/create_batch_custom_events.js

```diff
@@ -105,7 +106,7 @@ async function main() {
     console.log(JSON.stringify(response));
   }
 
-  callCreateBatchCustomEvents();
+  await callCreateBatchCustomEvents();
```

Comment:
QQ: Will this need any changes in the function code? (In Python, we need to updates the function def use `await`)

Reply:
in this case there is no need to make changes as we already marked this function as asynchronous using keyword: **async** in the declaration